### DISCUSSION TITLE
Unfortunately, we weren't able to approve your educational discount request for @Fhehudj.

### DISCUSSION THREAD

**Discussion by Fhehudj** [community/community#143144] [GitHub Education]
### Hi everyone,

I have been trying to pass verification for three weeks, can you help with verification. I have a student card, a pass and generally all possible information that can help in verification, but I can’t do it. Please help, I will be very grateful 
![Screenshot_20241031_131930_Chrome](https://github.com/user-attachments/assets/491e45ba-df8b-4c5d-8e4b-e82336eb47e7)

**Comment by SyedAliJunaidSherazi** [comment]
Hi there,

I understand how frustrating it can be to have trouble with the verification process. Here are a few steps you can try to expedite your verification for the educational discount:

1. **Check Eligibility**: Ensure that your institution is listed in GitHub’s Education Program. You can find the list on their website.

2. **Submit Required Documents**: Make sure you’re submitting all necessary documents, including your student card and any other proof of enrollment (like a student ID or enrollment letter). High-quality scans or clear photos often help.

3. **Contact Support**: If you haven’t already, reach out to GitHub Support directly. Include all relevant information and documentation to speed up the process. Be sure to mention how long you’ve been waiting.

4. **Double-Check Email**: Ensure that you’re using the correct email associated with your GitHub account for any communications.

5. **Follow Up**: If you don’t hear back within a few days, it’s okay to send a polite follow-up email to reiterate your request.

I hope this helps! Good luck, and don’t hesitate to ask if you need more assistance :)
